FrontEnd/global.scss

```diff
@@ -0,0 +1,11 @@
+:root {
+    --gold-primary: #BB8F52;
+    --gold-accent: rgba(201, 170, 136, 30);
+
+    --white: #ffffff;
+    --black: #333333;
+}
+
+html, button {
+    font-family: 'Open Sans';
+}
```

Comment:
so, the "index.scss" was meant to be the "global", but you're right.
could you please;
 *move this chunk in there
 *tidy that file up
 *rename that to be "global.scss",  and update the `routed.index.js` accordingly
 *also just noticed, that this file should be imported to the storybook header as well (currently missing) (either config.js (prefered) or the .html file)